crypto.go

```diff
@@ -156,6 +156,10 @@ type certificate struct {
 	Key  string
 }
 
+func getCertificate(cert string, key string) certificate {
```

Comment:
This is clever. I like this. imo, it's a good way to construct an object that's compatible with the existing `generateSignedCert(...)` function.

I'm not sure about the name `getCertificate`. The sentiment "get certificate" feels ambiguous. Is there some more accurate, more descriptive moniker we can give this? Something like `buildCustomCertificate()` maybe?

idk. Naming things is hard.